src/markup/codemirror/create.ts

```diff
@@ -145,6 +149,28 @@ export function createCodemirror(params: CreateCodemirrorParams) {
             scroll(event) {
                 onScroll(event);
             },
+            paste(event, editor) {
+                if (event.clipboardData && parseInsertedUrlAsImage) {
+                    const {imageUrl, title} =
+                        parseInsertedUrlAsImage(
+                            event.clipboardData.getData(DataTransferType.Text) ?? '',
+                        ) || {};
+
+                    if (!imageUrl) {
+                        return;
+                    }
+
+                    event.preventDefault();
+
+                    insertImages([
```

Comment:
Note: As far as I understand, when inserting this image, its size may exceed the size of the container, which may cause empty areas to appear on the screen. As a result, users have two different experiences: when inserting a file normally, the image fits into the specified dimensions, and when inserting a link from above, there may be empty space. Perhaps in the future it is worth preloading the image to correctly determine its size.